examples/general/dubbo/go-server/app/user_provider2.go

```diff
@@ -21,9 +21,7 @@ import (
 	"context"
 	"fmt"
 	"strconv"
-)
 
-import (
 	"github.com/apache/dubbo-go-hessian2/java_exception"
```

Comment:
format package. it should be three parts. the first is standard package, the second is third party package, and the third is `github.com/apache/dubbo-go/...`.